sample-code/examples/php/SimpleTest.php

```diff
@@ -20,9 +20,9 @@ class SimpleTest extends Sauce\Sausage\WebDriverTestCase
             'port' => 4723,
             'browserName' => '',
             'desiredCapabilities' => array(
-                'device' => 'iPhone Simulator',
-                'version' => '6.0',
-                'platform' => 'Mac',
+                'deviceName' => '=iPhone 5s',
```

Comment:
yes, it should be there: https://discuss.appium.io/t/could-not-find-a-device-to-launch-due-to-device-string-mutilation/2319/6